CONTRIBUTING.md

```diff
@@ -245,13 +251,27 @@ If in doubt, you can always ask for guidance in the Pull Request or on
 Feel free to post a comment in the Pull Request to ping reviewers if you are
 awaiting an answer on something.
 
+Notes: if the reviewer mentions *nits* in their comments, that stands for
+"requests for small changes that are not essential". Usually these nits are
+related to coding styles. It's always a good idea to pay attention to the
+detail!
 
 ### Step 8: Landing
 
-Once your Pull Request has been reviewed and approved by at least one Node.js
-Collaborators (often by saying LGTM, or Looks Good To Me), and as long as
-there is consensus (no objections from a Collaborator), a
-Collaborator can merge the Pull Request . GitHub often shows the Pull Request as
+A Pull Request needs to be reviewed and approved by at least one Node.js
+Collaborators (often by saying LGTM, or Looks Good To Me) to get landed.
```

Comment:
That might be a good start.

I think we may want to add some text about the validity of approvals (either via 'LGTM' or Github's PR review) after changes were made since the approval. For example, when someone approves a PR using Github's mechanism, it will still show "Approved" even after someone pushes more changes after that approval is made, which can be misleading (at least at first glance).